Tasks/Common/webdeployment-common-v2/ziputility.ts

```diff
@@ -25,13 +48,32 @@ const extractUsingUnzip = async (fromFile: string, toDir: string) => {
         .exec();
 }
 
+const isMSBuildPackageZip = async (packageZipPath: string): Promise<boolean> => {
```

Comment:
Is this the logic you were talking about that is already getting used elsewhere?

Reply:
copied from here https://github.com/microsoft/azure-pipelines-tasks/blob/db54986645736fa3fbbc938608924577c7b6e13a/common-npm-packages/AzureRmDeploy-common/webdeployment-common/packageUtility.ts#L68